build-constraints.yaml

```diff
@@ -531,6 +530,7 @@ packages:
         - machines-directory
```

Comment:
I just noticed why you aren't getting pinged: it says `bitemyapp` instead of `@bitemyapp`. I'll fix that now.